tests/Umbraco.Tests.Integration/Umbraco.Examine.Lucene/UmbracoExamine/BackOfficeExamineSearcherTests.cs

```diff
@@ -0,0 +1,661 @@
+    }
+
+    protected override void CustomTestSetup(IUmbracoBuilder builder)
+    {
+        builder.Services.AddUnique<IServerMessenger, ContentEventsTests.LocalServerMessenger>();
+        builder
+            .AddNotificationHandler<ContentTreeChangeNotification,
+                ContentTreeChangeDistributedCacheNotificationHandler>();
+        builder.AddNotificationHandler<ContentCacheRefresherNotification, ContentIndexingNotificationHandler>();
+        builder.AddExamineIndexes();
+        builder.AddBackOfficeIdentity();
+        builder.Services.AddHostedService<QueuedHostedService>();
+    }
+
+    private IEnumerable<ISearchResult> BackOfficeExamineSearch(string query, int pageSize = 20, int pageIndex = 0)
+    {
+        long totalFound = long.MinValue;
+        return BackOfficeExamineSearcher.Search(query, UmbracoEntityTypes.Document,
+            pageSize, pageIndex, out totalFound, ignoreUserStartNodes: true);
+    }
+
+    private async Task SetupUserIdentity(string userId)
+    {
+        var identity =
+            await BackOfficeUserStore.FindByIdAsync(userId, CancellationToken.None);
+        await BackOfficeSignInManager.SignInAsync(identity, false);
+    }
+
+    private PublishResult CreateDefaultPublishedContent(string contentName)
+    {
+        var contentType = new ContentTypeBuilder()
+            .WithId(0)
+            .Build();
+        ContentTypeService.Save(contentType);
+
+        var content = new ContentBuilder()
+            .WithId(0)
+            .WithName(contentName)
+            .WithContentType(contentType)
+            .Build();
+        var createdContent = ContentService.SaveAndPublish(content);
+
+        // It seems like we need this timeout for the moment, otherwise our indexes would not be built in time and the test could fail.
+        Thread.Sleep(1000);
+
+        return createdContent;
+    }
+
+    private PublishResult CreateDefaultPublishedContentWithTwoLanguages(string englishNodeName, string danishNodeName)
+    {
+        string usIso = "en-US";
+        string dkIso = "da";
+
+        var langDa = new LanguageBuilder()
+            .WithCultureInfo(dkIso)
+            .Build();
+        LocalizationService.Save(langDa);
+
+        var contentType = new ContentTypeBuilder()
+            .WithId(0)
+            .WithContentVariation(ContentVariation.Culture)
+            .Build();
+        ContentTypeService.Save(contentType);
+
+        var content = new ContentBuilder()
+            .WithId(0)
+            .WithCultureName(usIso, englishNodeName)
+            .WithCultureName(dkIso, danishNodeName)
+            .WithContentType(contentType)
+            .Build();
+        var createdContent = ContentService.SaveAndPublish(content);
+        Thread.Sleep(1000);
+
+        return createdContent;
+    }
+
+    [Test]
+    public void Search_Published_Content_With_Empty_Query()
+    {
+        SetupUserIdentity(Constants.Security.SuperUserIdAsString);
```

Comment:
Since this is an async method the test should also be async and await the method 😄 otherwise we can get all kinds of race conditions in the future. This is the case in all the tests 😄